File: RepoTransBench/target_projects/Java/C#/1321928757_Concurrent-MulThread/public_tests/LockPerformancePublicTest.cs
using System.Threading;
using Xunit;

namespace PublicTests
{
    public class LockPerformancePublicTest
    {
        [Fact]
        public void TestReentrantLockLowContention()
        {
            object lockObj = new object();
            int sum = 0;
            int N = 50; // Different from original value
            for (int i = 0; i < N; i++)
            {
                lock (lockObj)
                {
                    sum += i;
                }
            }

            Assert.True(sum > 0);
        }
    }
}

File: RepoTransBench/target_projects/Java/Go/1321928757_Concurrent-MulThread/public_tests/lock_performance_public_test.go
package public_tests

import (
	"sync"
	"testing"
)

func TestReentrantLockLowContention(t *testing.T) {
	var lock sync.Mutex
	sum := 0
	N := 50
	for i := 0; i < N; i++ {
		lock.Lock()
		sum += i
		lock.Unlock()
	}
	if sum <= 0 {
		t.Errorf("expected sum > 0, got %d", sum)
	}
}